shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h

```diff
@@ -11,6 +11,8 @@
 #include "flutter/shell/common/variable_refresh_rate_reporter.h"
 #include "flutter/shell/common/vsync_waiter.h"
 
+extern NSString* const kCADisableMinimumFrameDurationOnPhoneKey;
```

Comment:
This should have a declaration comment. Maybe link to https://developer.apple.com/documentation/quartzcore/optimizing_promotion_refresh_rates_for_iphone_13_pro_and_ipad_pro#3885321 ?

Reply:
Good call - done.